accepted/0000-uyuni-microos-management.md

```diff
@@ -0,0 +1,142 @@
+- Feature Name: (fill with a unique ident, my_awesome_feature)
+- Start Date: (fill with today's date, YYYY-MM-DD)
+
+# Summary
+[summary]: #summary
+
+Improve the management of openSUSE MicroOS in Uyuni.
+
+# Motivation
+[motivation]: #motivation
+
+MicroOS, or the downstream SLE Micro, is a different operating system than e.g. SLES. It is based on the same tools and packages, with few additions on top. These additions are few, but completely change the operating model.
+
+The main idea is: **All changes go into a new btrfs snapshot. This new snapshot is "pending" until a reboot activates it.**
+
+Until now, we tried to hide the differences in Uyuni and relied on Salt to do the *right thing*. This strategy was easy for us to use, but it did not work well. We need to change the approach we take with transactional systems.
+
+# Detailed design
```

Comment:
Should we add info here about the changes that pertain to displaying packages? I understand that we want to also display what packages are in a new transaction, but not yet applied (or vice versa, what packages are removed, but still present in the currently-running snapshot). The details of this would go to a different squad, but they might be in scope for this RFC?